Sources/PinLayout+WrapContent.swift

```diff
@@ -101,7 +101,7 @@ extension PinLayout {
     }
 
     private func wrapContent(_ type: WrapType, padding: PEdgeInsets, _ context: Context) -> PinLayout {
-        let subviews = view.subviews
+        let subviews = view.subviews.filter { !$0.isHidden }
```

Comment:
Sorry, but this could create an important breaking change for those using this method and expecting it to wrap also hidden views. 

I would propose instead that you add a new `ViewFilter` parameter to all `wrapContent` methods

**Types.swift**
```
@objc public enum ViewFilter: Int {
    /// No filter, use all views
    case none
    /// Consider only views where `hidden` is false
    case visibleOnly
}
```


**PinLayout+WrapContent.swift**
```
    public func wrapContent(viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(.all, padding: PEdgeInsets(top: 0, left: 0, bottom: 0, right: 0), viewFilter: viewFilter, { return "wrapContent()" })
    }

    public func wrapContent(padding: CGFloat, viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(.all, padding: PEdgeInsets(top: padding, left: padding, bottom: padding, right: padding), viewFilter: viewFilter, { return "wrapContent(padding: \(padding)" })
    }
    
    public func wrapContent(padding: PEdgeInsets, viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(.all, padding: padding, viewFilter: viewFilter, { return "wrapContent(padding: \(insetsDescription(padding))" })
    }

    public func wrapContent(_ type: WrapType, viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(type, padding: PEdgeInsets(top: 0, left: 0, bottom: 0, right: 0), viewFilter: viewFilter, { return "wrapContent(\(type.description)" })
    }

    public func wrapContent(_ type: WrapType, padding: CGFloat, viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(type, padding: PEdgeInsets(top: padding, left: padding, bottom: padding, right: padding), viewFilter: viewFilter, { return "wrapContent(\(type.description), padding: \(padding)" })
    }

    public func wrapContent(_ type: WrapType, padding: PEdgeInsets, viewFilter: ViewFilter = .none) -> PinLayout {
        return wrapContent(type, padding: padding, viewFilter: viewFilter, { return "wrapContent(\(type.description), padding: \(insetsDescription(padding))" })
    }

    private func wrapContent(_ type: WrapType, padding: PEdgeInsets, viewFilter: ViewFilter, _ context: Context) -> PinLayout {
   // TODO: Apply ViewFilter if not .none
   ....
```

Reply:
Thanks for the feedback, please review the changes